### PR TITLE
SD-4847: Update monitoring dashboard items when an item is moved

### PR DESCRIPTION
**Pull request by mscam**
* Update monitoring items when an item is sent to another stage/desk.
* Fixes related to #861 PR code review.

### CODE REVIEW

File: scripts/superdesk-desks/desks.js [hunk 1]
```diff
@@ -172,22 +172,22 @@
...
                 });
 
-                scope.$on('item:move', function($event, data) {
-                    if ((data.to_stage && data.to_stage === scope.stage) ||
-                        (data.from_stage && data.from_stage === scope.stage)) {
-                        scheduleQuery();
+                scope.$on('item:move', function(_e, data) {
+                    if ((data.to_desk && data.from_desk !== data.to_desk) ||
+                        (data.to_stage && data.from_stage !== data.to_stage))  {
+                        scheduleQuery(1000); // smaller delay.
```
Comment (gbbr) [inline scripts/superdesk-desks/desks.js]: You forgot to address @petrjasek's comment in https://github.com/superdesk/superdesk-client-core/pull/861#discussion_r75461689

**Comment by gbbr** [comment]
This is much better, thank you! I'm closing the other one.

File: scripts/superdesk-monitoring/monitoring.spec.js [hunk 2]
```diff
@@ -176,12 +176,13 @@ describe('monitoring', function() {
...
+
+            scope.$broadcast('item:move', {from_stage: 'bar', to_stage: 'bar'});
             scope.$digest();
             $timeout.flush(500);
             expect(api.query).not.toHaveBeenCalled();
 
-            scope.$broadcast('item:move', {to_stage: 'foo'});
+            scope.$broadcast('item:move', {from_stage: 'bar', to_stage: 'foo'});
             scope.$digest();
             $timeout.flush(1000);
```
Comment (petrjasek) [inline scripts/superdesk-monitoring/monitoring.spec.js]: you might need to increase this flush too

**Comment by gbbr** [comment]
LGTM

**Comment by gbbr** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

**Pull request merged**